### PR TITLE
[Data object] Advanced relation disable options

### PR DESCRIPTION
**Pull request by kjkooistra-youwe**
Disable delete, add and create options for advanced many to many (object) relation field via field configuration. This is useful when the records are added programmatically and therefore shouldn't be added or removed, but the metadata should still be editable.

### CODE REVIEW

File: bundles/CoreBundle/Resources/translations/nl.extended.json [hunk 1]
```diff
@@ -613,6 +613,9 @@
     "right": "right",
     "provide_split_view": "Provide Split View",
     "allow_multiple_assignments": "Allow Multiple Assignments",
+    "disable_delete": "Verwijderen uitschakelen",
+    "disable_add": "Toevoegen uitschakelen",
+    "disable_create": "Aanmaken uitschakelen",
```
Comment (youwe-petervanderwal) [inline bundles/CoreBundle/Resources/translations/nl.extended.json]: Do we need to add it to the other languages as well?

**Review by youwe-petervanderwal — APPROVED** [review]

File: bundles/CoreBundle/Resources/translations/nl.extended.json (hunk 1 above)
Comment (kjkooistra-youwe) [reply bundles/CoreBundle/Resources/translations/nl.extended.json]: Looks like they automatically fall back to English.

Comment (brusch) [reply bundles/CoreBundle/Resources/translations/nl.extended.json]: Translations please only for EN, other languages are managed at POEditor.

**Comment by brusch** [comment]
Thanks a lot for your PR! 
I'd like to leave this PR open a bit and discuss the actual need out there - just to make sure we're not adding more and more features that are very niche and are actually just making it harder to maintain the product. 

Just one more thing that comes to my mind ... if we decide to merge this PR, the settings should be consistent for all relational data-types - it seems some are actually missing, right?

**Comment by kjkooistra-youwe** [comment]
@brusch It's specifically added for the **advanced** many to many relation fields, since they contain metadata fields which can be edited. There is no way to disable adding or deleting records without also disabling the metadata fields. For regular relation fields the 'not editable' option usually is sufficient. It can be expanded to those fields as well of course.

**Comment by brusch** [comment]
Since there were no notable reactions on this and that there's a merge conflict, I'll close this PR for now. Please feel free to continue the discussion here 😊

**Comment by kjkooistra-youwe** [comment]
@brusch Will it be merged if I fix the merge conflict? Or is a discussion mandatory? And if so, who should be involved?

**Comment by kjkooistra-youwe** [comment]
I updated the branch, but I guess the issue won't update when closed.

**Comment by kjkooistra-youwe** [comment]
@brusch Can you please answer my previous question?

**Comment by brusch** [comment]
Since there were not notable demand for that feature, I'd leave it closed for now - just to keep things easier to maintain.

**Comment by kjkooistra-youwe** [comment]
@brusch That's fine, but who should be involved for further discussion on whether the feature is useful or not?

**Comment by brusch** [comment]
Normally people are upvoting or commenting on features, this a a good indicator for us how big the interest is 😊

**Comment by kjkooistra-youwe** [comment]
I guess this may be a very specific use case.

**Pull request closed without merging**